app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -1,12 +1,14 @@
 import React, { useState } from "react";
 
 import { Formik, Form } from "formik";
-import { Button, Pane } from "neetoui/v2";
-import { Input, Textarea } from "neetoui/v2/formik";
+import { Button, Pane, Select } from "neetoui/v2";
+import { Input } from "neetoui/v2/formik";
 
 import notesApi from "apis/notes";
 import formValidationSchemas from "constants/formValidationSchemas";
 
+import { SELECT_OPTIONS } from "./constants";
+
 export default function NoteForm({ onClose, refetch, note, isEdit }) {
```

Comment:
Let's restructure this. 
Please refrain from using `Pane, Form, Modal, Page` etc in component, filenames or folder names as suffixes or prefixes. Here, it can be named `Edit` or `Create` based on the usecase.

Reply:
@karthiknmenon Ideally what should one name a form component? For example, if we have CreatArticle.jsx and EditArticle.jsx both of which use the same form component, what should this component be named? ArticleForm.jsx?

Reply:
Ideally, `Form.jsx` would be what handles all form-related logic. The point is not prefix or suffix file name or component names with either `Article` or the other common suffixes. In a use-case where you're using a single component to render both forms conditionally, it makes sense to name it `Form.jsx`.

Reply:
Alright. Thanks!